LoopBack/LBFile.h

```diff
@@ -61,9 +74,9 @@ typedef void (^LBFileDownloadSuccessBlock)();
  * @param  localPath  The local path to the file, without file name.
  * @param  container  The file's container.
  */
-- (LBFile *)createFileWithName:(NSString*)name
-                     localPath:(NSString*)localPath
-                     container:(NSString*)container;
+- (LBFile *)createFileWithName:(NSString *)name
+                     localPath:(NSString *)localPath
+                     container:(NSString *)container;
```

Comment:
I don't usually make changes for format adjustments but decided to do some for this file since I'm touching many lines and consistent format helps readability.